PowerSlice/lang/PowerSlice.xml

```diff
@@ -8,7 +8,7 @@
           <multipleoptions>Create content</multipleoptions>
           <singleoptionprefix>New</singleoptionprefix>
         </createbutton>
-				<sortbutton>Sort</sortbutton>
+        <sortbutton>Sort</sortbutton>
```

Comment:
Since the whitespace issue was introduced in the same PR, would you mind squashing the two commits to one? Thanks in advance!